src/main/java/org/protege/editor/owl/client/diff/ui/ReviewButtonsPanel.java

```diff
@@ -244,7 +251,20 @@ public void actionPerformed(ActionEvent e) {
         	 
         }
     };
-    
+
+    private ActionListener restartPelletBtnListener = new ActionListener() {
+        @Override
+        public void actionPerformed(ActionEvent actionEvent) {
+            ClientSession sess = ClientSession.getInstance(editorKit);
+            LocalHttpClient client = (LocalHttpClient) sess.getActiveClient();
+
+            if (!client.isWorkFlowManager(sess.getActiveProject())) return;
+
+            // TODO: implement this
+//            pelletService.restart();
```

Comment:
Actually as I look at this code more closely I'm wondering if it would be cleaner to have `restart reasoner` added to the reasoner menu instead of another button in the revision panel. 

That would seem to fit better and be more like `pause server` is in the server menu.

With respect to the actual service restart call, it would be nice to not have to make the revision panel dependent on the pellet client. It already is dependent on the protege client, perhaps adding a method there might work better?